src/Context/CachedContextFactory.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Roave\BetterReflection\Context;
```

Comment:
If the entire source is used, would it be OK to drop the `$namespace` parameter?